pkg/agent/openflow/network_policy.go

```diff
@@ -698,9 +704,40 @@ func (c *client) UninstallPolicyRuleFlows(ruleID uint32) error {
 	return nil
 }
 
+func (c *client) reconcilePolicyFlows() {
```

Comment:
Would you consider handle this the same as reconciliation after Agent restarts, like to let NetworkPolicyController resync every rule?
@tnqn: do you have a good idea?

I am fine with this PR's approach, but like to understand if there could be simpler way, rather than adding extra code in the openflow client layer.

Reply:
We can use a channel (reuse the PodUpdates channel like nil value representing full sync or a separate channel) to notify networkpolicycontroller to resync all rules. But openflow client layer still needs to take care of cleaning its cache?

Reply:
Isn't it better to keep all that logic hidden from the controllers and handle it behind the scenes in the client? We also avoid unnecessary recomputation of flows and we can replay all the flows really fast "in one go" once we have the bundle support. If in the future we have a version of OVS that does not discard flows on restart, we can also remove this code.

Reply:
Yes, I think your current approach of replaying flows together is better than invoking upper layer separately.